.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libssh2"]
```

Comment:
Would rather not use submodules. It adds quite a bit of overhead on the developer and can be confusing to work with. The source code is included in the repo so as not to have to use submodules.

Reply:
I'd prefer to use git submodules as they're designed to do this very thing for tracking a particular remote's branch.
In this PR I've told it to track the 1.9.0 release commit from libssh2's tags/release on github.

All this needs is some work on putting hit hooks into the repo with some minor bash scripts. (If libssh2 ever get around to releasing another version that is)